src/hotspot/share/jfr/utilities/jfrJavaLog.cpp

```diff
@@ -28,7 +28,10 @@
 #include "jfr/utilities/jfrLogTagSets.hpp"
 #include "logging/log.hpp"
 #include "logging/logConfiguration.hpp"
+#include "logging/logMessage.hpp"
 #include "memory/resourceArea.hpp"
+#include "oops/objArrayOop.hpp"
+#include "oops/objArrayOop.inline.hpp"
```

Comment:
Same here.

Reply:
I ran into issue with building (without precompiled headers on Linux), if I didn't include "#include "oops/objArrayOop.hpp" in jfrJavaLog.cpp. Maybe there is some better way around this.

Reply:
I looked into this, and there is a circular dependency in objArrayOop.inline.hpp. I have created a PR to take care of that:
https://github.com/openjdk/jdk/pull/3431

Testing different build configurations with this now.